ui/app/components/workspace-switcher.hbs

```diff
@@ -37,7 +37,7 @@
               >
                 <span class="workspace-switcher__link-contents">
                   {{#if (eq workspace.name @current)}}
-                    <Pds::Icon @type="check-plain" />
+                    <FlightIcon @name="check-plain" />
```

Comment:
Doesn’t look like these quite came through (also not sure what’s happened to the alignment):

<img width="139" alt="CleanShot 2021-12-09 at 10 08 29@2x" src="https://user-images.githubusercontent.com/34030/145376467-f54085f8-590b-4f42-bc4b-6cdd274fd2b1.png">